routes/claims/claims.controller.js

```diff
@@ -0,0 +1,50 @@
+const { validationResult, checkSchema } = require('express-validator')
```

Comment:
The `claims.controller` is handling the `/deductions` routes, and the `deductions.controller` is doing the opposite. Can we just have one controller (deductions) that handles them all?

Reply:
ya, not sure how I missed that one